homeassistant/components/voice_assistant/pipeline.py

```diff
@@ -478,3 +499,98 @@ async def validate(self):
 
         if prepare_tasks:
             await asyncio.gather(*prepare_tasks)
+
+
+class PipelineStore:
+    """Class to hold a collection of pipelines."""
+
+    def __init__(self, hass: HomeAssistant) -> None:
+        """Initialize the pipeline store."""
+        self.hass = hass
+        self.pipelines: dict[str, Pipeline] = {}
+        self._store: Store[dict[str, Any]] = Store(
+            hass, STORAGE_VERSION, STORAGE_KEY, atomic_writes=True
```

Comment:
No need for atomic writes, this is not mission critical.

Reply:
Fixed